common/log/class.SingleFileAppender.php

```diff
@@ -169,7 +169,7 @@ public function doLog(common_log_Item $item)
                 '%t' => $item->getDateTime(),
                 '%r' => $item->getRequest(),
                 '%f' => $item->getCallerFile(),
-                '%g' => implode(',', $item->getTags()),
```

Comment:
- [ ] This is unrelated to the merge. Somebody forgot to release proper generis version